### PR TITLE
Removed unnecessary request from paginating funuction.

### PR DESCRIPTION
**Pull request by ashvardanian**

### CODE REVIEW

**Comment by ashvardanian** [comment]
It's nice to see a working library for Instagram, but the code is not very well organized. I have created a new version of `Instagram` and placed it into "Insta.py". It has a uniform API for all methods that support pagination.

File: igramscraper/instagram.py [hunk 1]
```diff
@@ -650,14 +658,13 @@ def get_paginate_medias(self, username, max_id=''):
             return to_return
 
         for mediaArray in nodes:
+            print('received raw:', mediaArray['node'])
```
Comment (NMelis) [inline igramscraper/instagram.py]: use print to logging is not good idea

File: igramscraper/insta.py [hunk 2]
```diff
@@ -0,0 +1,1150 @@
+import time
```
Comment (NMelis) [inline igramscraper/insta.py]: Why need this file? it's exists like igramscraper/instagram.py

File: examples/get_media_likes.py [hunk 3]
```diff
@@ -1,18 +1,25 @@
...
 
-# ...
+# # ...
 
 
+from context import Insta # pylint: disable=no-name-in-module
 
+instagram = Insta()
+# likes = instagram.get_media_likes_page('BG3Iz-No1IZ')
+# print(likes)
```
Comment (NMelis) [inline examples/get_media_likes.py]: commented code - why did you don't delete them

**Comment by realsirjoe** [comment]
I am closing this now since you don't seem to be interested in merging but just in promoting your own fork

**Comment by ashvardanian** [comment]
@realsirjoe Didn’t receive any notifications from here, so just continued with my own branch. Now I see that there were some changes in your repo since then

**Comment by ashvardanian** [comment]
@NMelis Agree with both statements. It looked like no one was interested in the pull request, so I simply continued updating the main branch. Regarding the use of `Instagram.py` it was fully abandoned in favor of the new file. I left it for compatibility purposes.

**Pull request closed without merging**